### PR TITLE
Maplayers widget: downgraded core to 3.7.2

### PR DESCRIPTION
**Pull request by mdastous-bentley**

### CODE REVIEW

**Review by aruniverse — COMMENTED** [review]
the presentation team added a test app in the monorepo. mind adding your pkg to it? take a look at the readme

**Comment by mdastous-bentley** [comment]
Currently block on this error when running test-viewer:
`caught Error: Multiple @itwin/core-frontend imports detected!`

**Comment by saskliutas** [comment]
You need to [inject](https://pnpm.io/package_json#dependenciesmetainjected) your widget package in test-viewer. Otherwise your package will be symlinked with it's devDependencies and they will clash with test-viewer dependencies.

Injecting package breaks webpack hot reloading but there is a script to work around that https://github.com/iTwin/viewer-components-react/blob/master/apps/test-viewer/README.md#running-app-with-workspace-packages-hot-reloading

**Comment by mdastous-bentley** [comment]
> You need to [inject](https://pnpm.io/package_json#dependenciesmetainjected) your widget package in test-viewer. Otherwise your package will be symlinked with it's devDependencies and they will clash with test-viewer dependencies.
> 
> Injecting package breaks webpack hot reloading but there is a script to work around that https://github.com/iTwin/viewer-components-react/blob/master/apps/test-viewer/README.md#running-app-with-workspace-packages-hot-reloading

@saskliutas I injected map-layers in the test-viewer. that solved the previous multiple imports error, however the hot reload doesnt seem to work, even after updating `linkWorkspaceDeps.js`

**Comment by mdastous-bentley** [comment]
> 1. Build your branch and launched test-viewer with `npm run start:dev`

Thanks works as expected now! I was using  `npm run start` instead of  `npm run start:dev`

File: common/changes/@itwin/map-layers/maplayers-core37_appui4_2023-04-18-12-36.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/map-layers",
+      "comment": "Adjusted map-layers dependencies (now use core packages 3.7.1)",
```
Comment (aruniverse) [inline common/changes/@itwin/map-layers/maplayers-core37_appui4_2023-04-18-12-36.json:5]: Set the comment to an empty string

File: packages/itwin/map-layers/src/ui/widget/MapLayerManager.scss [hunk 2]
```diff
@@ -268,7 +268,7 @@ $default-font-size: --iui-font-size-1;
 
     &:focus {
       outline: none;
-      box-shadow: $uicore-button-focus-boxshadow;
+      box-shadow: $buic-focus-boxshadow 0px 0px 0px 2px;
```
Comment (aruniverse) [inline packages/itwin/map-layers/src/ui/widget/MapLayerManager.scss:271]: Is there an itwin UI equivalent?

**Review by saskliutas — APPROVED** [review]
Approving test-viewer part

**Review by aruniverse — APPROVED** [review]
Check if there's an itwinui equivalent for the scss vars

Change the comment

File: packages/itwin/map-layers/src/ui/widget/MapLayerManager.scss (hunk 2 above)
Comment (mdastous-bentley) [reply packages/itwin/map-layers/src/ui/widget/MapLayerManager.scss]: @aruniverse merged jason's changes, so should be good now

**Pull request merged**